cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonUtils.java

```diff
@@ -304,5 +306,22 @@ public static Long getTimeInstant(String metadata) {
     public static boolean isMAdeOfAlphaNumericsOrUnderscores(String s) {
         return PATTERN.matcher(s).matches();
     } // isMAdeOfAlphaNumericsOrUnderscores
+    
```

Comment:
I guess this method was copied from `NGSIUtils.java`... it should be removed from there, since now it is a utility function of `CommonUtils.java`. Of course, several other source filw will be affected by this change.